TypeCobol.LanguageServer/Commands/AdjustFillers.cs

```diff
@@ -0,0 +1,156 @@
+                        var levelNumber = lastChild.CodeElement.LevelNumber;
+                        Debug.Assert(levelNumber != null);
+
+                        // Detect level and indentation from last child
+                        string level = levelNumber.Token.SourceText;
+                        int indent = levelNumber.Token.StartIndex;
+                        string newText = $"{Environment.NewLine}{new string(' ', indent)}{level} FILLER PIC X({adjustedFillerSize}).";
+
+                        // Insert right after last child
+                        int line = lastChild.CodeElement.LineEnd;
+                        int column = lastChild.CodeElement.StopIndex + 1;
+                        var start = new Position(line, column);
+                        var end = new Position(line, column);
+                        textEdit = new TextEdit(new VsCodeProtocol.Range(start, end), newText);
+                    }
+
+                    if (textEdit != null)
+                    {
+                        textEdits.Add(textEdit);
+                    }
+                }
+
+                void Remove()
+                {
+                    // Remove the whole code element (including spaces at the beginning of the line)
+                    var start = new Position(lastChild.CodeElement.Line, 0);
+                    var end = new Position(lastChild.CodeElement.LineEnd, lastChild.CodeElement.StopIndex + 1);
+                    var textEdit = new TextEdit(new VsCodeProtocol.Range() { start = start, end = end }, string.Empty);
+                    textEdits.Add(textEdit);
+                }
+            }
+        }
+
+        public static AdjustFillers Create(TypeCobolServer typeCobolServer) => new AdjustFillers(typeCobolServer);
+
+        public AdjustFillers(TypeCobolServer typeCobolServer)
+            : base(typeCobolServer)
+        {
+
+        }
+
+        public override object Run(object[] arguments)
+        {
+            if (arguments == null || arguments.Length == 0 || !TryReadArgumentAs(arguments[0], out TextDocumentIdentifier textDocumentIdentifier))
+            {
+                throw new ArgumentException("Invalid arguments for command.", nameof(arguments));
+            }
+
+            var target = Server.GetDocumentContextFromStringUri(textDocumentIdentifier.uri, Workspace.SyntaxTreeRefreshLevel.RebuildNodes);
+            var programClassDocument = target.FileCompiler?.CompilationResultsForProgram?.ProgramClassDocumentSnapshot;
+            if (programClassDocument == null)
+            {
+                throw new InvalidOperationException($"Could not get AST for document '{target.Uri}'.");
+            }
+
+            // Compute edits asynchronously: create a message for ourselves
+            Server.Workspace.MessagesActionsQueue.Enqueue(new MessageActionWrapper(() => ComputeTextEdits(target.Uri, programClassDocument)));
+            return null;
```

Comment:
As discussed, it would be better to return `new object()` so that the client is able to make a difference between a successful command and an error.

Reply:
done